samples/file-input-example/README.md

```diff
@@ -0,0 +1,39 @@
+# Example: Send logs from file to Sumo Logic
+
+This example sends logs from `logs.txt` file into your Sumo HTTP source.
+
+## Prerequisites
+
+Download the latest plugin's release into current directory.
+
+```bash
+curl -s https://api.github.com/repos/SumoLogic/logstash-output-sumologic/releases/latest |
+  grep browser_download_url |
+  cut -d '"' -f 4 |
+  wget -qi -
```

Comment:
Can we stick to `curl` instead of using both `curl` and `wget`?

Reply:
Done